src/cook-web/src/components/outline-tree/index.tsx

```
@@ -0,0 +1,231 @@
+                    <span className='flex flex-row items-center w-40 whitespace-nowrap overflow-hidden text-ellipsis' >
+                        <InlineInput
+                            isEdit={focusId === props.item.id}
+                            value={cataData[props.item.id!]?.name || ""}
+                            onChange={onNodeChange}
+                            onFocus={() => {
+                                actions.setFocusId(props.item.id || "")
+                            }}
+                        />
+                    </span>
+                    {
+                        (props.item?.depth || 0 > 0) && (
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)